rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+# RFC002 - Blog & video content publishing model (aka "Procope")
+
+## Proposed By
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X)) , Carrie Crowe ([@Carrie Crowe](https://thegooddocs.slack.com/team/U01QS8YAHHN))
+
+## Proposal State
+
+- [x] Draft (submitted to GitHub on May 26, 2021)
+- [ ] Under discussion 
+- [ ] Final comment and voting ({{date}})
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal Overview
+
+Producing and hosting blog posts and video content, including publication ownership and models.
+
+### Motivation
+
+Part of promoting good documentation is helping those uncertain about (or overwhelmed by) what constitutes good docs and best practices. We seek to create a space where the community can contribute to the body of knowledge, as it relates to the mission of the Good Docs Project.
+
+We also want to foster community contributors, to elevate voices that aren't elevated and to empower individuals to be thought leaders in their careers.
+
+## Proposal Details
+
+Regarding specifics on dates and number of participants, this proposal represents our blue sky ideal. It's intended to be flexible with reality, but for simplicity's sake we won't keep mentioning that as a caveat throughout this document.
+
+### Scope of proposal
+
+This proposal focuses on "community-driven media," namely **blog posts** and **videos** about individual topics, created by individual community members and based on those individual viewpoints. All these media elements will be curated by the editorial collective, steered by the core publishers (and input from the Project Steering Committee).
+
+The audiences for this media are:
+* Developers and other "non-technical writers" looking to empower themselves, either in using our templates or in other applicable writing advice
+* Technical writers in that same position
+There is no expectation that either audience will also be GDP contributors or community members. Material will always shoot for outside of the contributor sphere.
+
+#### Out of scope
+
+This proposal doesn't cover **white papers**, though it does address some nuance between white papers and blog posts for clarity.
+
+This proposal also doesn't cover the potential for other media like **podcasts**, but that could be a future expansion.
+
+This proposal also doesn't cover social media, but is certainly tethered to that element of community engagement.
+
+#### Aside: defining white papers vs blog posts
```

Comment:
thought (non-blocking): In my mind, a "blog post" is something that is written as the need arises, as the author things of something interesting. I wonder whether it should be called something else. I'm afraid I don't have a good suggestion,  "Periodical" doesn't sound right either.

Reply:
That isn't my experience with blogs. Some people treat them as a repo of one-offs, many others as periodicals. That's what I would put in the white paper category.